Makefile

```diff
@@ -54,32 +54,32 @@ docker: build
 
 	cp $(BUILD_DIR)/api api
 	chmod 755 api/api
-	docker build api -t sodafoundation/multi-cloud-api:latest
+	docker build api -t sodafoundation/multi-cloud-api:v9
```

Comment:
Why v9 tag? Images can be latest or with release tag

Reply:
V9 : this image is built with some custom changes for go-micro deployment in Kubernetes.
will be reverted to latest once the code is merged and stabilitzed.